action.yml

```diff
@@ -1,56 +1,151 @@
 # Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
-name: 'Connect Tailscale'
-description: 'Connect your GitHub Action workflow to Tailscale'
+name: "Connect Tailscale"
+description: "Connect your GitHub Action workflow to Tailscale"
 branding:
-  icon: 'arrow-right-circle'
-  color: 'gray-dark'
+  icon: "arrow-right-circle"
+  color: "gray-dark"
 inputs:
   authkey:
-    description: 'Your Tailscale authentication key, from the admin panel.'
+    description: "Your Tailscale authentication key, from the admin panel."
     required: true
-  version:
-    description: 'Tailscale version to use.'
-    required: true
-    default: '1.24.2'
   args:
-    description: 'Optional additional arguments to `tailscale up`'
+    description: "Optional additional arguments to `tailscale up`"
     required: false
+    deprecationMessage: "Please use specific input, this is no longer used in the script"
     default: ''
+  version:
+    description: "Tailscale version to use."
+    required: true
+    default: "1.30.0"
```

Comment:
Since this PR was opened, the default has already been advanced to `1.32.1`. The version here should be at least that high.